### PR TITLE
added a blog

### PR DESCRIPTION
**Pull request by suruozhong**
Fixes # .

Make sure these boxes are checked before creating your pull request -- thank you!

- [x] Add Blog Header, including title, author, date.
e.g.

``` md
---
 title: "'Score Your Face Photo' a ML&Wechaty practice"
 date: 2017-09-18 09:00 +0800
 author: huyingxi
 ---
 > Author: [@huyingxi](https://github.com/huyingxi/wechaty_selfie) enjoying ML&Wechaty at BUPT
```

- [x] Keep all filenames & url as lowercase, and use `-` to connect words instead of space. e.g. `2017-10-06-wechat-pc-impactor` instead of `2017-10-06-WeChat PC Impactor`,
- [x] Embed the photo & video before publishing, save all external file to the blog `/download/2018` directory.
- [x] Add `<!--more-->` section to just show the abstract in blog homepage.
- [x] Add teaser image for the blog
- [x] Add your self as an author to the author data file at <https://github.com/wechaty/wechaty.github.io/blob/master/_data/authors.yml> with your GitHub username. And please also remember to reference yourself with the Github username in your blog post, so that we can have nice author information besides your blog after we published it!

[See more](https://github.com/wechaty/wechaty.github.io#contribute-guideline)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
```
Comment (huan) [inline package.json]: Please remove the unrelated file.

**Review by huan — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (suruozhong) [reply package.json]: Deleted

File: _posts/2020-05-20-wechaty-team-robot.md [hunk 2]
```diff
@@ -0,0 +1,228 @@
+---
+title: "团队小助手"
+author: suruozhong
+date: '2020-05-20 12:00:00 +0800'
+categories: project
+tags:
```
Comment (lijiarui) [inline _posts/2020-05-20-wechaty-team-robot.md:6]: Please add a teaser(image) here to make the blog more beauriful

**Review by lijiarui — CHANGES_REQUESTED** [review]
Please change as I request.

File: _posts/2020-05-20-wechaty-team-robot.md (hunk 2 above)
Comment (suruozhong) [reply _posts/2020-05-20-wechaty-team-robot.md:6]: added

**Review by lijiarui — APPROVED** [review]

**Review by lijiarui — APPROVED** [review]

**Comment by lijiarui** [comment]
Congratulations! 🎉 🎉 🎉

You become a wechaty contributor successfully! 👏 👏 👏

Please:

- [ ]  Add my wechat: ruirui_0914 and send me this pr link. I will invite you into Wechaty Contributor Room
- [ ]  Join wechaty org by clicking https://github.com/wechaty

image

**Pull request merged**